### DISCUSSION TITLE
Is there an example of how i can add a .dylib and swift for iOS

### DISCUSSION THREAD

**Discussion by gaithoben** [callstack/react-native-builder-bob#293] [Q&A]
I would like to make function calls that are in a .dylib file

after a week, i cant figure it out.

Anyone who can help with an example?

**Comment by berendsliedrecht** [comment]
I can not offer you a way how, but calling a dylib within an application will make it impossible to push to the App Store.

Only applications developed by Apple can make calls to a dylib.
